user-manuals/orders/bopis/appnotifications/README.md

```diff
@@ -1,6 +1,6 @@
 # Orders Page
 
-Orders page reflects the journey of an order from receiving the pickup order via ecommerce platform till the order is complete.
+The Orders page can be accessed by clicking on the `Orders` button on the bottom tab of the BOPIS app. It displays a list of all the orders currently assigned to the facility for store pickup from the e-commerce platform, and it can be used to search, view and fulfill orders. This page reflects the journey of an order from receiving the pickup order via ecommerce platform till the order is complete.
```

Comment:
We need to mention that the default page that is opened in the BOPIS app is the Orders Page

Reply:
Done.